content-src/components/DiscoveryStreamComponents/Navigation/_Navigation.scss

```diff
@@ -33,7 +33,7 @@
     a {
       &:hover {
         // text-decoration: underline; didn't quite match comps.
-        border-bottom: 1px solid $blue-40;
+        border-bottom: 1px solid var(--newtab-link-primary-color);
```

Comment:
Is there a reason you wouldn't use a direct color var from `content-src/styles/_variables.scss` like we're doing for the bulk of DS?

Reply:
Because the links colour is set to `color: var(--newtab-link-primary-color);` already.

Reply:
Hmm. Ok, that probably should be a Sass var as well, but since we're already inconsistent I guess it doesn't hurt.

Reply:
What color var would you use from _variables.scss ?

Reply:
We need to resolve that mismatch, but yeah, I don't think it's a fight for this bug.

Reply:
Hopefully it can be a discussion and not a fight, but agree it's something we need to figure out a consistent approach toward.